### PR TITLE
AST-448 - fix: Multiselect close icon not showing on selected items.

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Multiselect close icon not showing on selected items in customizer.
- https://wp-astra.atlassian.net/browse/AST-448

### Screenshots
- https://share.getcloudapp.com/Kou4ym85

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js [hunk 1]
```diff
@@ -2039,6 +2039,7 @@ S2.define('select2/selection/multiple',[
       }
 
       $selection.text(formatted);
+      $selection.prepend('<span class="select2-selection__choice__remove" role="presentation" aria-hidden="true">&times; </span>');
```
Comment (imnavanath) [inline inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:2042]: IMO, ideally x sign needs to be appear after text - https://share.getcloudapp.com/Kou4ym85 (not prepend)

**Comment by imnavanath** [comment]
@sagarb90 changelog is missing here, can you please add too.

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js (hunk 1 above)
Comment (sagarb90) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:2042]: @imnavanath the default position for x icon in select2 library is before the text please refer this.
 https://share.getcloudapp.com/8LuoyLl2
 But if we want to do that we can do this here but in add-on we need to add custom CSS for that.

Comment (imnavanath) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:2042]: Okay, but UI wise it looks little bit odd. Select2 UI is different which suits to their icon position. This improvement may come from designer, better to confirm this once with Satish/Monika.

**Review by abhishek-bsf — APPROVED** [review]

File: inc/customizer/custom-controls/assets/js/unminified/custom-controls.js (hunk 1 above)
Comment (imnavanath) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:2042]: @sagarb90 any update on this?

Comment (sagarb90) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:2042]: @imnavanath Yes, we had asked Monika and move the cross button after the text, also added comments in the task by Monika.

Comment (imnavanath) [reply inc/customizer/custom-controls/assets/js/unminified/custom-controls.js:2042]: Great 👍 Thanks.

File: changelog.txt [hunk 2]
```diff
@@ -2,6 +2,7 @@ v3.5.1 (Unreleased)
 - Fix: Max-width responsive CSS enlarges logo size on mobile devices in old header layout.
 - Fix: Cart Icon not working with Sticky Header after enabling the option "Hide when scrolling down" on mobile devices.
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
+- Improvement: Added close button for multiselect selected items in google font variants.
```
Comment (premanshup) [inline changelog.txt:5]:
```suggestion
- Improvement: Added close button for multiselect selected items in Google font variants.
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 2 above)
Comment (sagarb90) [reply changelog.txt]: Done

**Review by premanshup — APPROVED** [review]

**Pull request merged**